### ISSUE TITLE
Single file deployment

### ISSUE THREAD

**Issue by leonardder** [obiwanjacobi/vst.net#43]
**Is your feature request related to a problem? Please describe.**
A difference between VSt's created with vst.net and other vsts I've seen is that a deployment of a vst.net project requires a significant amount of files (e.g. assembly dlls).

**Describe the solution you'd like**
In .net 5.0, it seems that single application/single file deployment was improved a lot. Could this be an area to explore further?

**Comment by obiwanjacobi** [comment]
Yes, definitely. When .NET 5 is released I will look how I can improve vstnet deployment and single file deployment is certainly on the wish list. To be honest, I was very pleased to get this working as it is ;-)

**Comment by leonardder** [comment]
I looked into single file deployment briefly, but was a bit disappointed to see that it doesn't seem to work for class libraries. Whatever you do, the output is a *.nupkg for them

**Comment by aminya** [comment]
I managed to fix the above using ILRepack:
https://github.com/ravibpatel/ILRepack.Lib.MSBuild.Task

Here is the config file needed for that. The VST.NET files need to stay outside
https://github.com/robianmcd/midi-shape-shifter/blob/master/MidiShapeShifter/MidiShapeShifter/ILRepack.targets
